scripts/run_examples.py

```diff
@@ -86,11 +89,20 @@ def run_examples():
     if errored:
         print('Some examples failed!')
         print('Assets saved in: %s' % output_dir)
-        sys.exit(1)
+        return True, tested_files_full_paths
     else:
         print('All examples ran successfully!')
-        rm_rf(output_dir)
+        return False, tested_files_full_paths
```

Comment:
Ooops

Reply:
Note it didn't do it before either. `output_dir` was being reassigned for each example, so only the last one was removed. Since they are temporary directories on a CI, should we cleanup? That would require moving the artifacts on each loop. No strong opinion here, just wondering!

Reply:
I think it should work now, more easily too.